scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -144,11 +178,12 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
       libraries,
       outputPath.toIO,
       mainClass,
+      legacy,
       testBridgeInit,
-      mode == FullOpt,
+      optimize,
```

Comment:
Since `FullOpt` and `FastOpt` were used only internally I deprecated them now.
We can delete them when we release Mill 0.11